File: portal/portal/App_Code/PaginaAcesso.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace portal.App_Code
{
    public class PaginaAcesso
    {
        private double _codigo;
        private string _nome;
        private string _descritivo;
        private string _pagina;
        private TipoStatus _status;

        public double Codigo
        {
            get
            {
                return _codigo;
            }

            set
            {
                _codigo = value;
            }
        }

        public string Nome
        {
            get
            {
                return _nome;
            }

            set
            {
                _nome = value;
            }
        }

        public string Descritivo
        {
            get
            {
                return _descritivo;
            }

            set
            {
                _descritivo = value;
            }
        }

        public string Pagina
        {
            get
            {
                return _pagina;
            }

            set
            {
                _pagina = value;
            }
        }

        public TipoStatus Status
        {
            get
            {
                return _status;
            }

            set
            {
                _status = value;
            }
        }

        public enum TipoStatus { Ativo, Inativo };


    }
}

File: portal/portal/App_Code/PaginaAcesso.php
<?php
###################################################################################################################
/*PÁGINA AECESSO: Gerencia área administrativa
Ex: Descritivo, Posts, Implementa controle de acesso dos usuários, cadastrar novo usuário, Notícia, etc.*/
####################################################################################################################

namespace portal\App_Code {
//ATRIBUTOS----------------------------------------------------------------------------------------------------------		
    class PaginaAcesso {
        private $codigo;
        private $nome;
        private $descritivo;
        private $pagina;
        private $TipoStatus; //1 - Ativo, 0 - Inativo.
//CONSTRUTOR----------------------------------------------------------------------------------------------------------			
		public function __construct (){}
        
        public function setPaginaAcesso($codigo,$nome,$descritivo,$pagina,$TipoStatus){
			$this -> codigo = $codigo;
			$this -> nome = $nome;
			$this -> descritivo = $descritivo;
			$this -> pagina = $pagina;
			$this -> TipoStatus = $TipoStatus;
		}
//SETS-----------------------------------------------------------------------------------------------------------------        
		public function setCodigo($codigo){
			$this -> codigo = $codigo;
        }
        public function setNome($nome) {
			$this -> nome = $nome;
        }
		public function setDescritivo($descritivo) {
			$this -> descritivo = $descritivo;
        }
		Public function setPagina($pagina) {
			$this -> pagina = $pagina;
        }
        public function setTipoStatus($TipoStatus) {
			$this -> TipoStatus = $TipoStatus;
        }
//GETS-----------------------------------------------------------------------------------------------------------------			
		public function getCodigo(){
			return $this -> codigo;
        }
        public function getNome(){
			return $this -> nome;
        }
        public function getDescritivo(){
			return $this -> descritivo;
        }	
        public function getPagina(){
			return $this -> pagina;
        }
        public function getTipoStatus() {
			return $this -> TipoStatus;
        }
    }
}

?>